code/exceptions/RealMeException.php

```diff
@@ -0,0 +1,12 @@
+<?php
+class RealMeException extends Exception
+{
+    const INVALID_IDENTITY_VALUE = 0;
```

Comment:
@dhensby subclasses feels a bit overkill just for some structure around status codes, do you have a standard we could refer to?